Fix link to Application Insights for Service Fabric

fbeltrao opened the pull request:
The link to Microsoft Application Insights for Service Fabric is pointing to a non-existing page:
https://github.com/Microsoft/ApplicationInsights-ServiceFabric/blob/develop/README.md

Updating to use the master branch:
https://github.com/Microsoft/ApplicationInsights-ServiceFabric/blob/master/README.md

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@4c74356b41 
Another good catch! Thank you!
--Carolyn

cjgronlund commented:
@barbkess

fbeltrao committed:
Fix link to Application Insights for Service Fabric

Current URL: https://github.com/Microsoft/ApplicationInsights-ServiceFabric/blob/develop/README.md
Proposed URL: https://github.com/Microsoft/ApplicationInsights-ServiceFabric/blob/master/README.md
